src/app/services/hris/employee/employee-profile.service.ts

```diff
@@ -29,6 +30,11 @@ export class EmployeeProfileService {
     return this.httpClient.get<SimpleEmployee>(`${this.baseUrl}/simple-profile${queryParams}`);
   }
 
+  getEmployeeProfileDetailsById(id: number): Observable<EmployeeProfileDetails> {
+    const queryParams = `?id=${id}`;
+    return this.httpClient.get<EmployeeProfileDetails>(`${environment.HttpsBaseURL}/profile-details${queryParams}`);
```

Comment:
Refactor: check the other methods, they should all look similar, use this.baseUrl

Reply:
fixed